config/mail.php

```diff
@@ -15,7 +15,7 @@
     |
     */
 
-    'driver' => env('MAIL_DRIVER', 'smtp'),
+    'driver' => env('MAIL_DRIVER', 'mandrill'),
```

Comment:
I know! but just felt like we should default to what we are expecting to use...i mean i doesn't matter i guess cause these things should always be set